client/signup/steps/new-or-existing-site/index.tsx

```diff
@@ -40,11 +46,14 @@ export default function NewOrExistingSiteStep( props: Props ): React.ReactNode {
 	const headerText = translate( 'Do It For Me' );
 
 	const subHeaderText = translate(
-		'Get a professionally designed, mobile-optimized website in %(fulfillmentDays)d business days or less for a one-time fee of {{PriceWrapper}}%(displayCost)s{{/PriceWrapper}} plus a one year subscription of the Premium plan.',
+		'Get a professionally designed, mobile-optimized website in %(fulfillmentDays)d business days or less for a one-time fee of {{PriceWrapper}}%(displayCost)s{{/PriceWrapper}} plus a one year subscription of the %(plan)s plan.',
 		{
 			args: {
 				displayCost,
 				fulfillmentDays: 4,
+				plan: isEnabled( 'plans/pro-plan' )
```

Comment:
Nit. Normally it reads better import all and do. It provides context quickly that this is a feature flag coming from the config. Just a personal style preference, feel free to ignore it 😁 .
```suggestion
				plan: config.isEnabled( 'plans/pro-plan' )
```
```suggestion
				plan: isEnabled( 'plans/pro-plan' )
```

Reply:
TIL! Good suggestion. However, since this is a super temporary change that will likely be removed in a few days, let's keep it as it is. I'll keep this in mind for next time though!